jaeger-2.yaml

```diff
@@ -147,3 +149,10 @@ test:
         timeout: 60
         expected_output: |
           Admin server started
+
+update:
+  enabled: true
+  github:
+    use-tag: true
+    identifier: jaegertracing/jaeger
+    strip-prefix: v
```

Comment:
Can we also pin the version prefix to `2.` to pick always given version suffix in the package name.